src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -2356,18 +2356,48 @@ FixedwingPositionControl::control_manual_position(const float control_interval,
 	attitude_setpoint.copyTo(_att_sp.q_d);
 }
 
-void FixedwingPositionControl::control_backtransition(const float control_interval, const Vector2f &ground_speed,
-		const position_setpoint_s &pos_sp_curr)
+void FixedwingPositionControl::control_backtransition_heading_hold()
 {
-	const bool is_low_height = checkLowHeightConditions();
```

Comment:
Did you on purpose remove the is_low_height check?

Reply:
@sfuhrer Yes, we don't run TECS during transitions anymore, as we don't use either pitch or throttle.